public/main/main.pug

```diff
@@ -2,30 +2,34 @@
+        .small(style="display: inline; font-size: 24px; color: #bbb") Created by Giovanna Ceserani and Giorgio Caviglia
         .small Explore the journeys and the lives of British and Irish travelers to eighteenth-century Italy
-      button.btn.btn-lg.btn-success.btn-lavender(ui-sref="search") Start exploring 
+      button.btn.btn-lg.btn-lavender(ui-sref="search") Start exploring 
        i.fa.fa-angle-right
 
 .container.main-view
 
   .row
     .col-lg-12
 
-      p.text-serif A digital project created by Giovanna Ceserani and Giorgio Caviglia at CESTA Stanford University
+      p.text-serif A digital tool created and developed by Giovanna Ceserani and Giorgio Caviglia at CESTA (the Center for Spatial and Textual Analysis at Stanford University), with development contributions by Cody Leff, Ashwin Ramaswami and Ryan Tan.
       
-      h3 The Grand Tour Explorer is a digital transformation of John Ingamells’ 
-        em A Dictionary of British and Irish Travellers in Italy 1701-1800 
-        | (published for the 
-        em Paul Mellon Centre for Studies in British Art 
-        | by 
-        em Yale University Press, 
-        | 1997). It transforms this text into a research tool that facilitates multidimensional searching and browsing of the entries in Ingamells’ Dictionary; the organization of this information into structured and usable data; and the analysis and visualization of this data.
-
-
-      p.text-serif The tool is still under construction; private beta version currently being tested with the support of Malcolm Baker, John Brewer, Melissa Calaresu, Jeffrey Collins, Paul Davis, Christopher Johns, Simon Macdonald, Carole Paul, Sophus Reinert, Catherine Sama, Rosemary Sweet.
-
-
-
-      p.text-serif For information about the project, please contact Giovanna Ceserani at 
-        a(href="mailto:ceserani@stanford.edu") ceserani@stanford.edu
+      h3 The Grand Tour Explorer is a dynamic and interactive resource for the exploration of eighteenth-century British travel to Italy. At its core is the digital transformation of the 5232 entries in 
+        em A Dictionary of British and Irish Travelers in Italy 1701-1800 
+        | (compiled by John Ingamells from the Brinsley Ford Archives at the Paul Mellon Center in London, YUP 1997). The dictionary's alphabetized print form is recast into a multidimensional digital database about the lives and journeys of more than 6000 travelers. As a research tool, the Grand Tour Explorer facilitates searching and browsing this mass of information; it organizes this information into structured and usable data; and it enables the analysis and visualization of this data.
+
+
+      p.text-serif This is an open access digital resource: you can explore freely, save the results of your explorations, and download the data for your further research. Learn more about the 
+        span.hover-item history 
+        | of this project and about the 
+        span.hover-item rationale 
```

Comment:
These hover items seem like they should be links. Are we going to add links to them?

![image](https://user-images.githubusercontent.com/1689183/66275897-3b300880-e842-11e9-9546-9198f4714c8f.png)

Reply:
I believe that was the intention, I'm just building off of @ceserani's mock-up